pkgs/tools/text/grex/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "grex";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "pemistahl";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "141bf7lhclpxdgjzvznq7ns4kplxbfkjq03rx91b95dch6c6bfl4";
+  };
+
+  cargoSha256 = "1l6vwdib0wxag8hvsjhqmy2n0ri15qf9drnimppcg3qcpz3mlga0";
```

Rebase onto origin/master, then:

```suggestion
  cargoSha256 = "02chy973na34ncc648v5hfrh1l48qjpbnx3qnkdig7k4pz29ynyj";
```